src/Apps/Sell/Utils/formUtils.ts

```diff
@@ -1,11 +1,15 @@
 export const PROVENANCE_LIST = [
-  "Purchased directly from gallery",
-  "Purchased directly from artist",
-  "Purchased at auction",
-  "Gift from the artist",
-  "Other",
-  "I don’t know",
-].map(provenance => ({
-  value: provenance,
-  text: provenance,
-}))
+  { value: "", text: "Select" },
```

Comment:
To avoid the duplications, we could also use mal and do something like

text: value || "Select"

Reply:
🙈 you're right